encrypted-payloads/crypt-converter.go

```diff
@@ -0,0 +1,150 @@
+package cryptconverter
```

Comment:
I don't like package name is not the same as dir name. Move this to `cryptconverter` sub-dir and remove package alias in the worker and starter.